src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -141,15 +147,16 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setName(toCopy.name);
             setPhone(toCopy.phone);
             setEmail(toCopy.email);
-            setAddress(toCopy.address);
+            setTelegram(toCopy.handle);
+            setGitHub(toCopy.username);
             setTags(toCopy.tags);
         }
 
         /**
          * Returns true if at least one field is edited.
          */
         public boolean isAnyFieldEdited() {
-            return CollectionUtil.isAnyNonNull(name, phone, email, address, tags);
+            return CollectionUtil.isAnyNonNull(name, phone, email, handle, tags);
```

Comment:
I think we should add github `username` here or else the user cannot edit just the github field.
```suggestion
            return CollectionUtil.isAnyNonNull(name, phone, username, email, handle, tags);
```

Reply:
Agreed. Added 👍